File: problem_1/problem_1.cpp
#include <stdio.h>


int main(){
		int toplam=0;
	
	for(int i=0; i<1000; i++){
		
		if(i%3==0){
			toplam +=i;
		}
		else if(i%5==0){
			toplam +=i;
		}
		
}
	
	printf("Toplam sonucu : %d",toplam);
	
	return 0;
}

File: problem_1/problem_1.dart
void main(List<String> args) {
  var toplam = 0;
  for (int i = 0; i < 1000; i++) {
    if (i % 3 == 0) {
      toplam += i;
    } else if (i % 5 == 0) {
      toplam += i;
    }
  }
  print("Toplamın sonucu $toplam");
}
